src/components/EventManagement/AllEvents.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from "react";
+import { NavLink } from "react-router-dom";
+import { getEventdata } from "../../api";
+const AllEvents = () => {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const jsonData = await getEventdata();
+      console.log("json data get", jsonData);
```

Comment:
remove console log

Reply:
no need to create a new async  function in useEffect. Please  assign useEffect function as async or use the Promise.then() approach to implement the logic.

Reply:
removed